tests/integration/api/test_feature_flags.py

```diff
@@ -45,9 +44,17 @@ def test_feature_flags_override_flags(admin_client):
             {"condition": "boolean", "value": False},
         ],
     },
+    FEATURE_SOME_PLATFORM_FLAG_ENABLED=True,
```

Comment:
It does not seem to be a real flag, isn't?

Reply:
no, just for testing